lib/game_manager.ex

```diff
@@ -0,0 +1,59 @@
+defmodule GameManager do
+
+  def start(continue \\ true) when continue do
+    board = Board.empty
+    
+    board = play(board)
+
+    GameIO.print_board(board)
+
+    if Game.check_draw(board) do
+      IO.puts("Cats game!")
```

Comment:
It's nice that GameIO is abstracted out, and decouples some UI logic from this game loop, but `IO.puts` works against us on that.